Not Found: /api/v1/sentiment/
"POST /api/v1/sentiment/ HTTP/1.1" 404 2308
Bad Request: /api/sentiment/
"POST /api/sentiment/ HTTP/1.1" 400 2
Bad Request: /api/sentiment/
"POST /api/sentiment/ HTTP/1.1" 400 2
Bad Request: /api/sentiment/
"POST /api/sentiment/ HTTP/1.1" 400 89
Internal Server Error: /api/sentiment/
Traceback (most recent call last):
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/django/views/generic/base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/garnetsharara/IntelliAfrica/SentimentAnalysisApi/api/views.py", line 29, in post
    'sentiment': self.sentiment.get_sentiment()
  File "/home/garnetsharara/IntelliAfrica/SentimentAnalysisApi/services/sentiment.py", line 41, in get_sentiment
    return settings.MODEL.predict(self.tokenize())
  File "/home/garnetsharara/IntelliAfrica/SentimentAnalysisApi/services/sentiment.py", line 36, in tokenize
    tokenizer = Tokenizer()
NameError: name 'Tokenizer' is not defined
"POST /api/sentiment/ HTTP/1.1" 500 97570
Internal Server Error: /api/sentiment/
Traceback (most recent call last):
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/django/views/generic/base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/garnetsharara/IntelliAfrica/SentimentAnalysisApi/api/views.py", line 29, in post
    'sentiment': self.sentiment.get_sentiment()
  File "/home/garnetsharara/IntelliAfrica/SentimentAnalysisApi/services/sentiment.py", line 42, in get_sentiment
    return settings.MODEL.predict(self.tokenize())
  File "/home/garnetsharara/IntelliAfrica/SentimentAnalysisApi/services/sentiment.py", line 38, in tokenize
    tokenizer.fit_on_texts([preprocess(x) for x in self.X])
  File "/home/garnetsharara/IntelliAfrica/SentimentAnalysisApi/services/sentiment.py", line 38, in <listcomp>
    tokenizer.fit_on_texts([preprocess(x) for x in self.X])
NameError: name 'preprocess' is not defined
"POST /api/sentiment/ HTTP/1.1" 500 101418
Internal Server Error: /api/sentiment/
Traceback (most recent call last):
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/django/views/generic/base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/garnetsharara/IntelliAfrica/SentimentAnalysisApi/api/views.py", line 26, in post
    return JsonResponse(
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/django/http/response.py", line 603, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
  File "/usr/lib64/python3.9/json/__init__.py", line 234, in dumps
    return cls(
  File "/usr/lib64/python3.9/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib64/python3.9/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/home/garnetsharara/.local/lib/python3.9/site-packages/django/core/serializers/json.py", line 105, in default
    return super().default(o)
  File "/usr/lib64/python3.9/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type ndarray is not JSON serializable
"POST /api/sentiment/ HTTP/1.1" 500 123185
Internal Server Error: /api/v1/sentiment/
Traceback (most recent call last):
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/tino/Projects/Sentiment/Trained models/SentimentAnalysisApi/api/views.py", line 26, in post
    self.sentiment = self.sentiment(model=settings.MODEL, sentence=payload.validated_data['text'])
TypeError: Sentiment() got multiple values for argument 'model'
"POST /api/v1/sentiment/ HTTP/1.1" 500 108616
Method Not Allowed: /api/v1/sentiment/
"GET /api/v1/sentiment/ HTTP/1.1" 405 40
Not Found: /
"GET / HTTP/1.1" 404 2443
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2494
Not Found: /portal
"GET /portal HTTP/1.1" 404 2479
Internal Server Error: /portal/dashboard/
Traceback (most recent call last):
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/base.py", line 119, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/list.py", line 154, in get
    self.object_list = self.get_queryset()
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/list.py", line 36, in get_queryset
    raise ImproperlyConfigured(
django.core.exceptions.ImproperlyConfigured: Home is missing a QuerySet. Define Home.model, Home.queryset, or override Home.get_queryset().
"GET /portal/dashboard/ HTTP/1.1" 500 95681
Method Not Allowed (GET): /portal/dashboard/
Method Not Allowed: /portal/dashboard/
"GET /portal/dashboard/ HTTP/1.1" 405 0
Method Not Allowed (GET): /portal/dashboard/
Method Not Allowed: /portal/dashboard/
"GET /portal/dashboard/ HTTP/1.1" 405 0
Method Not Allowed (GET): /portal/dashboard/
Method Not Allowed: /portal/dashboard/
"GET /portal/dashboard/ HTTP/1.1" 405 0
Internal Server Error: /portal/dashboard/
Traceback (most recent call last):
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/base.py", line 119, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/list.py", line 154, in get
    self.object_list = self.get_queryset()
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/list.py", line 36, in get_queryset
    raise ImproperlyConfigured(
django.core.exceptions.ImproperlyConfigured: Home is missing a QuerySet. Define Home.model, Home.queryset, or override Home.get_queryset().
"GET /portal/dashboard/ HTTP/1.1" 500 95681
Not Found: /portal/dashboard/assets/images/flags/us.svg
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar7.jpg
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/xs/avatar7.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
Not Found: /portal/dashboard/assets/images/xs/avatar1.png
Not Found: /portal/dashboard/assets/images/flags/bl.svg
Not Found: /portal/dashboard/assets/images/themes/dark.png
Not Found: /portal/dashboard/assets/images/themes/rtl.png
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar1.png HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar8.jpg
Not Found: /portal/dashboard/assets/images/flags/es.svg
Not Found: /portal/dashboard/assets/images/xs/avatar9.jpg
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar9.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar8.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar6.jpg
"GET /portal/dashboard/assets/images/xs/avatar6.jpg HTTP/1.1" 404 2666
"GET /static/assets/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 404 1909
Not Found: /portal/dashboard/favicon.ico
"GET /portal/dashboard/favicon.ico HTTP/1.1" 404 2615
Not Found: /portal/dashboard/assets/images/themes/rtl.png
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/themes/dark.png
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar1.png
"GET /portal/dashboard/assets/images/xs/avatar1.png HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/us.svg
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar7.jpg
Not Found: /portal/dashboard/assets/images/flags/es.svg
Not Found: /portal/dashboard/assets/images/xs/avatar6.jpg
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/xs/avatar6.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar7.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/bl.svg
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar8.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar9.jpg
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar8.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar9.jpg HTTP/1.1" 404 2666
"GET /static/assets/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 404 1909
Not Found: /portal/dashboard/assets/images/xs/avatar8.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar9.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar6.jpg
"GET /portal/dashboard/assets/images/xs/avatar8.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar9.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar6.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/jp.svg
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
Not Found: /portal/dashboard/assets/images/flags/bl.svg
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/us.svg
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/es.svg
Not Found: /portal/dashboard/assets/images/themes/rtl.png
Not Found: /portal/dashboard/assets/images/xs/avatar7.jpg
"GET /portal/dashboard/assets/images/xs/avatar7.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/themes/dark.png
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar1.png
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
"GET /portal/dashboard/assets/images/xs/avatar1.png HTTP/1.1" 404 2666
"GET /static/assets/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 404 1909
Not Found: /portal/dashboard/favicon.ico
"GET /portal/dashboard/favicon.ico HTTP/1.1" 404 2615
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/flags/us.svg
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/themes/rtl.png
Not Found: /portal/dashboard/assets/images/flags/es.svg
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/themes/dark.png
Not Found: /portal/dashboard/assets/images/themes/default.png
Not Found: /portal/dashboard/assets/images/xs/avatar1.png
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/xs/avatar1.png HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar7.jpg
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
"GET /portal/dashboard/assets/images/xs/avatar7.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar6.jpg
Not Found: /portal/dashboard/assets/images/flags/bl.svg
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar9.jpg
"GET /portal/dashboard/assets/images/xs/avatar9.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar8.jpg
"GET /portal/dashboard/assets/images/xs/avatar8.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar6.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
"GET /static/assets/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 404 1909
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/themes/rtl.png
Not Found: /portal/dashboard/assets/images/themes/dark.png
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
Not Found: /portal/dashboard/assets/images/flags/us.svg
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
Not Found: /portal/dashboard/assets/images/flags/es.svg
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar1.png
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar6.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar7.jpg
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/xs/avatar6.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar1.png HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/bl.svg
"GET /portal/dashboard/assets/images/xs/avatar7.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar9.jpg
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar9.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar8.jpg
"GET /portal/dashboard/assets/images/xs/avatar8.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
Not Found: /portal/dashboard/assets/images/themes/dark.png
Not Found: /portal/dashboard/assets/images/flags/us.svg
Not Found: /portal/dashboard/assets/images/themes/rtl.png
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
Not Found: /portal/dashboard/assets/images/flags/es.svg
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar6.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar1.png
Not Found: /portal/dashboard/assets/images/xs/avatar7.jpg
"GET /portal/dashboard/assets/images/xs/avatar1.png HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar6.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar7.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar9.jpg
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/xs/avatar9.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar8.jpg
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/bl.svg
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar8.jpg HTTP/1.1" 404 2666
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/themes/default.png
Not Found: /portal/dashboard/assets/images/themes/dark.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
Not Found: /portal/dashboard/assets/images/themes/rtl.png
Not Found: /portal/dashboard/assets/images/flags/es.svg
Not Found: /portal/dashboard/assets/images/flags/bl.svg
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/us.svg
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
Not Found: /portal/dashboard/assets/images/flags/jp.svg
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar1.png
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar1.png HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar7.jpg
"GET /portal/dashboard/assets/images/xs/avatar7.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar6.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar8.jpg
"GET /portal/dashboard/assets/images/xs/avatar6.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar9.jpg
"GET /portal/dashboard/assets/images/xs/avatar8.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar9.jpg HTTP/1.1" 404 2666
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/themes/default.png
Not Found: /portal/dashboard/assets/images/themes/dark.png
Not Found: /portal/dashboard/assets/images/themes/rtl.png
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/bl.svg
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/us.svg
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
Not Found: /portal/dashboard/assets/images/flags/es.svg
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/flags/jp.svg
Not Found: /portal/dashboard/assets/images/themes/dark.png
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
Not Found: /portal/dashboard/assets/images/themes/rtl.png
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/us.svg
Not Found: /portal/dashboard/assets/images/flags/es.svg
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/bl.svg
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/us.svg
Not Found: /portal/dashboard/assets/images/themes/rtl.png
Not Found: /portal/dashboard/assets/images/themes/dark.png
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/es.svg
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
Not Found: /portal/dashboard/assets/images/flags/bl.svg
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
Internal Server Error: /admin/portal/project/
Traceback (most recent call last):
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/backends/django.py", line 62, in render
    return self.template.render(context)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/base.py", line 1000, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/base.py", line 958, in render_annotated
    return self.render(context)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/base.py", line 1000, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/base.py", line 958, in render_annotated
    return self.render(context)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/base.py", line 1000, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/base.py", line 958, in render_annotated
    return self.render(context)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/base.py", line 1000, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/base.py", line 958, in render_annotated
    return self.render(context)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/base.py", line 1000, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/base.py", line 958, in render_annotated
    return self.render(context)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/contrib/admin/templatetags/base.py", line 45, in render
    return super().render(context)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/template/library.py", line 260, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/contrib/admin/templatetags/admin_list.py", line 336, in result_list
    "results": list(results(cl)),
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/contrib/admin/templatetags/admin_list.py", line 312, in results
    yield ResultList(None, items_for_result(cl, res, None))
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/contrib/admin/templatetags/admin_list.py", line 303, in __init__
    super().__init__(*items)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/contrib/admin/templatetags/admin_list.py", line 224, in items_for_result
    result_repr = display_for_value(value, empty_value_display, boolean)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/contrib/admin/utils.py", line 448, in display_for_value
    return str(value)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/models/fields/reverse_related.py", line 127, in __repr__
    self.related_model._meta.app_label,
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/utils/functional.py", line 49, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/models/fields/reverse_related.py", line 91, in related_model
    if not self.field.model:
AttributeError: type object 'Users' has no attribute 'model'
"GET /admin/portal/project/ HTTP/1.1" 500 335540
Internal Server Error: /admin/portal/project/
Traceback (most recent call last):
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/backends/mysql/base.py", line 75, in execute
    return self.cursor.execute(query, args)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/MySQLdb/cursors.py", line 206, in execute
    res = self._query(query)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/MySQLdb/cursors.py", line 319, in _query
    db.query(q)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/MySQLdb/connections.py", line 254, in query
    _mysql.connection.query(self, query)
MySQLdb._exceptions.OperationalError: (1054, "Unknown column 'users_users.image' in 'field list'")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/contrib/admin/options.py", line 683, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/utils/decorators.py", line 133, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/decorators/cache.py", line 62, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/contrib/admin/sites.py", line 230, in inner
    if not self.has_permission(request):
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/contrib/admin/sites.py", line 203, in has_permission
    return request.user.is_active and request.user.is_staff
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/utils/functional.py", line 258, in inner
    self._setup()
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/utils/functional.py", line 398, in _setup
    self._wrapped = self._setupfunc()
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/contrib/auth/middleware.py", line 25, in <lambda>
    request.user = SimpleLazyObject(lambda: get_user(request))
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/contrib/auth/middleware.py", line 11, in get_user
    request._cached_user = auth.get_user(request)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/contrib/auth/__init__.py", line 198, in get_user
    user = backend.get_user(user_id)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/contrib/auth/backends.py", line 158, in get_user
    user = UserModel._default_manager.get(pk=user_id)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/models/query.py", line 492, in get
    num = len(clone)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/models/query.py", line 302, in __len__
    self._fetch_all()
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/models/query.py", line 1507, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/models/query.py", line 57, in __iter__
    results = compiler.execute_sql(
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1361, in execute_sql
    cursor.execute(sql, params)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/backends/utils.py", line 103, in execute
    return super().execute(sql, params)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/backends/mysql/base.py", line 75, in execute
    return self.cursor.execute(query, args)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/MySQLdb/cursors.py", line 206, in execute
    res = self._query(query)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/MySQLdb/cursors.py", line 319, in _query
    db.query(q)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/MySQLdb/connections.py", line 254, in query
    _mysql.connection.query(self, query)
django.db.utils.OperationalError: (1054, "Unknown column 'users_users.image' in 'field list'")
"GET /admin/portal/project/ HTTP/1.1" 500 206425
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
Not Found: /portal/dashboard/assets/images/themes/dark.png
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
Not Found: /portal/dashboard/assets/images/flags/bl.svg
Not Found: /portal/dashboard/assets/images/themes/rtl.png
Not Found: /portal/dashboard/assets/images/flags/us.svg
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
Not Found: /portal/dashboard/assets/images/flags/es.svg
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/jp.svg
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/favicon.ico
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
Not Found: /portal/dashboard/assets/images/flags/bl.svg
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/themes/dark.png
Not Found: /portal/dashboard/assets/images/themes/rtl.png
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
Not Found: /portal/dashboard/assets/images/flags/us.svg
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
Not Found: /portal/dashboard/assets/images/flags/es.svg
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/jp.svg
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/favicon.ico
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
Not Found: /portal/dashboard/assets/images/themes/dark.png
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/flags/es.svg
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/bl.svg
Not Found: /portal/dashboard/assets/images/themes/rtl.png
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/us.svg
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/jp.svg
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/favicon.ico
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
Not Found: /portal/dashboard/assets/images/themes/dark.png
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
Not Found: /portal/dashboard/assets/images/themes/rtl.png
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
Not Found: /portal/dashboard/assets/images/flags/us.svg
Not Found: /portal/dashboard/assets/images/flags/bl.svg
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/es.svg
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
Not Found: /favicon.ico
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/themes/dark.png
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
Not Found: /portal/dashboard/assets/images/flags/es.svg
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
Not Found: /portal/dashboard/assets/images/flags/us.svg
Not Found: /portal/dashboard/assets/images/themes/rtl.png
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/bl.svg
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
Internal Server Error: /portal/dashboard/
Traceback (most recent call last):
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/backends/mysql/base.py", line 75, in execute
    return self.cursor.execute(query, args)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/MySQLdb/cursors.py", line 206, in execute
    res = self._query(query)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/MySQLdb/cursors.py", line 319, in _query
    db.query(q)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/MySQLdb/connections.py", line 254, in query
    _mysql.connection.query(self, query)
MySQLdb._exceptions.OperationalError: (1054, "Unknown column 'portal_project.project_progress' in 'field list'")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/base.py", line 119, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/list.py", line 174, in get
    context = self.get_context_data()
  File "/home/tino/Projects/Sentiment/Trained models/SentimentAnalysisApi/portal/views.py", line 13, in get_context_data
    print(super(Home, self).get_context_data(**kwargs))
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/models/query.py", line 296, in __repr__
    data = list(self[: REPR_OUTPUT_SIZE + 1])
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/models/query.py", line 302, in __len__
    self._fetch_all()
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/models/query.py", line 1507, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/models/query.py", line 57, in __iter__
    results = compiler.execute_sql(
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1361, in execute_sql
    cursor.execute(sql, params)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/backends/utils.py", line 103, in execute
    return super().execute(sql, params)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/db/backends/mysql/base.py", line 75, in execute
    return self.cursor.execute(query, args)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/MySQLdb/cursors.py", line 206, in execute
    res = self._query(query)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/MySQLdb/cursors.py", line 319, in _query
    db.query(q)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/MySQLdb/connections.py", line 254, in query
    _mysql.connection.query(self, query)
django.db.utils.OperationalError: (1054, "Unknown column 'portal_project.project_progress' in 'field list'")
"GET /portal/dashboard/ HTTP/1.1" 500 170825
Not Found: /favicon.ico
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/themes/dark.png
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
Not Found: /portal/dashboard/assets/images/flags/bl.svg
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
Not Found: /portal/dashboard/assets/images/themes/rtl.png
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/us.svg
Not Found: /portal/dashboard/assets/images/flags/es.svg
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/favicon.ico
Not Found: /favicon.ico
Not Found: /favicon.ico
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
Not Found: /portal/dashboard/assets/images/themes/dark.png
Not Found: /portal/dashboard/assets/images/themes/rtl.png
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
Not Found: /portal/dashboard/assets/images/flags/es.svg
Not Found: /portal/dashboard/assets/images/flags/us.svg
Not Found: /portal/dashboard/assets/images/flags/bl.svg
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/favicon.ico
Not Found: /favicon.ico
Not Found: /favicon.ico
Not Found: /favicon.ico
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
Not Found: /portal/dashboard/assets/images/themes/dark.png
Not Found: /portal/dashboard/assets/images/flags/bl.svg
Not Found: /portal/dashboard/assets/images/themes/rtl.png
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
Not Found: /portal/dashboard/assets/images/flags/us.svg
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/es.svg
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/login.html
"GET /portal/dashboard/login.html HTTP/1.1" 404 2612
Not Found: /portal/dashboard/register.html
"GET /portal/dashboard/register.html HTTP/1.1" 404 2621
Not Found: /portal/dashboard/forgot-password.html
"GET /portal/dashboard/forgot-password.html HTTP/1.1" 404 2642
Not Found: /portal/dashboard/app-chat.html
"GET /portal/dashboard/app-chat.html HTTP/1.1" 404 2621
Not Found: /portal/dashboard/app-contact.html
"GET /portal/dashboard/app-contact.html HTTP/1.1" 404 2630
Not Found: /favicon.ico
Not Found: /portal/dashboard/favicon.ico
Not Found: /portal/dashboard/page-search.html
"GET /portal/dashboard/page-search.html HTTP/1.1" 404 2630
Not Found: /portal/dashboard/index-2.html
"GET /portal/dashboard/index-2.html HTTP/1.1" 404 2618
Not Found: /favicon.ico
Not Found: /portal/dashboard/favicon.ico
Not Found: /portal/dashboard/project-ticket.html
"GET /portal/dashboard/project-ticket.html HTTP/1.1" 404 2639
Not Found: /portal/dashboard/index-2.html
"GET /portal/dashboard/index-2.html HTTP/1.1" 404 2618
Internal Server Error: /api/v1/sentiment/
Traceback (most recent call last):
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/tino/Projects/Sentiment/Trained models/SentimentAnalysisApi/api/views.py", line 25, in post
    self.sentiment = Sentiment(settings.MODEL, payload.validated_data['text'])
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/conf/__init__.py", line 88, in __getattr__
    val = getattr(self._wrapped, name)
AttributeError: 'Settings' object has no attribute 'MODEL'
"POST /api/v1/sentiment/ HTTP/1.1" 500 98288
Internal Server Error: /api/v1/sentiment/
Traceback (most recent call last):
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/tino/Projects/Sentiment/Trained models/SentimentAnalysisApi/api/views.py", line 25, in post
    self.sentiment = Sentiment(settings.MODEL, payload.validated_data['text'])
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/conf/__init__.py", line 88, in __getattr__
    val = getattr(self._wrapped, name)
AttributeError: 'Settings' object has no attribute 'MODEL'
"POST /api/v1/sentiment/ HTTP/1.1" 500 98304
Not Found: /portal/dashboard/index-2.html
"GET /portal/dashboard/index-2.html HTTP/1.1" 404 2618
Not Found: /portal/dashboard/index-2.html
"GET /portal/dashboard/index-2.html HTTP/1.1" 404 2618
Not Found: /portal/dashboard/project-ticket.html
"GET /portal/dashboard/project-ticket.html HTTP/1.1" 404 2639
Not Found: /portal/dashboard/project-ticket-details.html
"GET /portal/dashboard/project-ticket-details.html HTTP/1.1" 404 2663
Not Found: /portal/dashboard/app-chat.html
"GET /portal/dashboard/app-chat.html HTTP/1.1" 404 2621
Not Found: /portal/dashboard/login.html
"GET /portal/dashboard/login.html HTTP/1.1" 404 2612
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2494
Not Found: /
"GET / HTTP/1.1" 404 2443
Not Found: /
"GET / HTTP/1.1" 404 2443
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/themes/dark.png
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
Not Found: /portal/dashboard/assets/images/themes/rtl.png
Not Found: /portal/dashboard/assets/images/themes/default.png
Not Found: /portal/dashboard/assets/images/flags/us.svg
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
Not Found: /portal/dashboard/assets/images/flags/jp.svg
Not Found: /portal/dashboard/assets/images/flags/es.svg
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/bl.svg
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
Not Found: /
"GET / HTTP/1.1" 404 2443
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
Not Found: /portal/dashboard/assets/images/flags/us.svg
Not Found: /portal/dashboard/assets/images/themes/rtl.png
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/themes/dark.png
Not Found: /portal/dashboard/assets/images/flags/es.svg
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
Not Found: /portal/dashboard/assets/images/flags/bl.svg
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
Not Found: /portal/dashboard/assets/images/themes/dark.png
Not Found: /portal/dashboard/assets/images/themes/rtl.png
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
Not Found: /portal/dashboard/assets/images/flags/es.svg
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/us.svg
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/bl.svg
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/login.html
"GET /portal/dashboard/login.html HTTP/1.1" 404 2612
Not Found: /portal/dashboard/page-profile.html
"GET /portal/dashboard/page-profile.html HTTP/1.1" 404 2633
Not Found: /portal/dashboard/project-ticket.html
"GET /portal/dashboard/project-ticket.html HTTP/1.1" 404 2639
Not Found: /portal/dashboard/project-ticket-details.html
"GET /portal/dashboard/project-ticket-details.html HTTP/1.1" 404 2663
Not Found: /portal/dashboard/index-2.html
"GET /portal/dashboard/index-2.html HTTP/1.1" 404 2618
Not Found: /portal/dashboard/app-chat.html
"GET /portal/dashboard/app-chat.html HTTP/1.1" 404 2621
Not Found: /portal/dashboard/project-ticket-details.html
"GET /portal/dashboard/project-ticket-details.html HTTP/1.1" 404 2663
Not Found: /portal/dashboard/project-ticket.html
"GET /portal/dashboard/project-ticket.html HTTP/1.1" 404 2639
Not Found: /portal/dashboard/app-chat.html
"GET /portal/dashboard/app-chat.html HTTP/1.1" 404 2621
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
Not Found: /portal/dashboard/assets/images/themes/rtl.png
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/themes/dark.png
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
Not Found: /portal/dashboard/assets/images/flags/es.svg
Not Found: /portal/dashboard/assets/images/flags/us.svg
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/bl.svg
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/themes/dark.png
Not Found: /portal/dashboard/assets/images/themes/rtl.png
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
Not Found: /portal/dashboard/assets/images/flags/es.svg
Not Found: /portal/dashboard/assets/images/themes/default.png
Not Found: /portal/dashboard/assets/images/flags/us.svg
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/bl.svg
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
Internal Server Error: /api/v1/sentiment/
Traceback (most recent call last):
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/tino/Projects/Sentiment/Trained models/SentimentAnalysisApi/api/views.py", line 25, in post
    self.sentiment = Sentiment(settings.MODEL, payload.validated_data['text'])
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/conf/__init__.py", line 88, in __getattr__
    val = getattr(self._wrapped, name)
AttributeError: 'Settings' object has no attribute 'MODEL'
"POST /api/v1/sentiment/ HTTP/1.1" 500 98304
Internal Server Error: /api/v1/sentiment/
Traceback (most recent call last):
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/tino/Projects/Sentiment/Trained models/SentimentAnalysisApi/api/views.py", line 25, in post
    self.sentiment = Sentiment(settings.MODEL, payload.validated_data['text'])
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/conf/__init__.py", line 88, in __getattr__
    val = getattr(self._wrapped, name)
AttributeError: 'Settings' object has no attribute 'MODEL'
"POST /api/v1/sentiment/ HTTP/1.1" 500 98304
Internal Server Error: /api/v1/sentiment/
Traceback (most recent call last):
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/tino/Projects/Sentiment/Trained models/SentimentAnalysisApi/api/views.py", line 25, in post
    self.sentiment = Sentiment(settings.MODEL, payload.validated_data['text'])
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/conf/__init__.py", line 88, in __getattr__
    val = getattr(self._wrapped, name)
AttributeError: 'Settings' object has no attribute 'MODEL'
"POST /api/v1/sentiment/ HTTP/1.1" 500 98304
Internal Server Error: /api/v1/sentiment/
Traceback (most recent call last):
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/tino/Projects/Sentiment/Trained models/SentimentAnalysisApi/api/views.py", line 25, in post
    self.sentiment = Sentiment(settings.MODEL, payload.validated_data['text'])
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/conf/__init__.py", line 88, in __getattr__
    val = getattr(self._wrapped, name)
AttributeError: 'Settings' object has no attribute 'MODEL'
"POST /api/v1/sentiment/ HTTP/1.1" 500 98304
Not Found: /portal/dashboard/app-chat.html
"GET /portal/dashboard/app-chat.html HTTP/1.1" 404 2621
Not Found: /
"GET / HTTP/1.1" 404 2443
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2494
Not Found: /
"GET / HTTP/1.1" 404 2443
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2494
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
Not Found: /portal/dashboard/assets/images/themes/rtl.png
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/themes/dark.png
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/flags/jp.svg
Not Found: /portal/dashboard/assets/images/flags/es.svg
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/bl.svg
Not Found: /portal/dashboard/assets/images/flags/us.svg
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/favicon.ico
Not Found: /portal/dashboard/assets/images/themes/rtl.png
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/themes/dark.png
Not Found: /portal/dashboard/assets/images/flags/es.svg
Not Found: /portal/dashboard/assets/images/flags/us.svg
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/bl.svg
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/app-chat.html
"GET /portal/dashboard/app-chat.html HTTP/1.1" 404 2621
Not Found: /portal/dashboard/app-contact.html
"GET /portal/dashboard/app-contact.html HTTP/1.1" 404 2630
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/themes/dark.png
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
Not Found: /portal/dashboard/assets/images/themes/rtl.png
Not Found: /portal/dashboard/assets/images/flags/us.svg
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/es.svg
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
Not Found: /portal/dashboard/assets/images/flags/bl.svg
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/page-profile.html
"GET /portal/dashboard/page-profile.html HTTP/1.1" 404 2633
Not Found: /portal/dashboard/assets/images/themes/rtl.png
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/flags/us.svg
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/themes/dark.png
Not Found: /portal/dashboard/assets/images/flags/es.svg
Not Found: /portal/dashboard/assets/images/flags/jp.svg
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/bl.svg
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/page-profile.html
"GET /portal/dashboard/page-profile.html HTTP/1.1" 404 2633
Not Found: /
"GET / HTTP/1.1" 404 2443
Not Found: /portal/dashboard/page-profile.html
"GET /portal/dashboard/page-profile.html HTTP/1.1" 404 2633
"GET /static/assets/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 404 1909
Internal Server Error: /api/v1/sentiment/
Traceback (most recent call last):
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/tino/Projects/Sentiment/Trained models/SentimentAnalysisApi/api/views.py", line 25, in post
    self.sentiment = Sentiment(settings.MODEL, payload.validated_data['text'])
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/conf/__init__.py", line 88, in __getattr__
    val = getattr(self._wrapped, name)
AttributeError: 'Settings' object has no attribute 'MODEL'
"POST /api/v1/sentiment/ HTTP/1.1" 500 96781
Internal Server Error: /api/v1/sentiment/
Traceback (most recent call last):
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/tino/Projects/Sentiment/Trained models/SentimentAnalysisApi/api/views.py", line 25, in post
    self.sentiment = Sentiment(settings.MODEL, payload.validated_data['text'])
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/conf/__init__.py", line 88, in __getattr__
    val = getattr(self._wrapped, name)
AttributeError: 'Settings' object has no attribute 'MODEL'
"POST /api/v1/sentiment/ HTTP/1.1" 500 96781
Internal Server Error: /api/v1/sentiment/
Traceback (most recent call last):
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/tino/Projects/Sentiment/Trained models/SentimentAnalysisApi/api/views.py", line 25, in post
    self.sentiment = Sentiment(settings.MODEL, payload.validated_data['text'])
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/conf/__init__.py", line 88, in __getattr__
    val = getattr(self._wrapped, name)
AttributeError: 'Settings' object has no attribute 'MODEL'
"POST /api/v1/sentiment/ HTTP/1.1" 500 96781
Internal Server Error: /api/v1/sentiment/
Traceback (most recent call last):
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/tino/Projects/Sentiment/Trained models/SentimentAnalysisApi/api/views.py", line 25, in post
    self.sentiment = Sentiment(settings.MODEL, payload.validated_data['text'])
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/conf/__init__.py", line 88, in __getattr__
    val = getattr(self._wrapped, name)
AttributeError: 'Settings' object has no attribute 'MODEL'
"POST /api/v1/sentiment/ HTTP/1.1" 500 96781
Internal Server Error: /api/v1/sentiment/
Traceback (most recent call last):
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/tino/Projects/Sentiment/Trained models/SentimentAnalysisApi/api/views.py", line 25, in post
    self.sentiment = Sentiment(settings.MODEL, payload.validated_data['text'])
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/conf/__init__.py", line 88, in __getattr__
    val = getattr(self._wrapped, name)
AttributeError: 'Settings' object has no attribute 'MODEL'
"POST /api/v1/sentiment/ HTTP/1.1" 500 96781
Internal Server Error: /api/v1/sentiment/
Traceback (most recent call last):
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/views/generic/base.py", line 84, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/tino/anaconda3/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/tino/Projects/Sentiment/Trained models/SentimentAnalysisApi/api/views.py", line 25, in post
    self.sentiment = Sentiment(settings.MODEL, payload.validated_data['text'])
  File "/home/tino/anaconda3/lib/python3.9/site-packages/django/conf/__init__.py", line 88, in __getattr__
    val = getattr(self._wrapped, name)
AttributeError: 'Settings' object has no attribute 'MODEL'
"POST /api/v1/sentiment/ HTTP/1.1" 500 96781
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
Not Found: /portal/dashboard/assets/images/themes/rtl.png
Not Found: /portal/dashboard/assets/images/themes/dark.png
Not Found: /portal/dashboard/assets/images/flags/us.svg
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/bl.svg
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/es.svg
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
"GET /static/assets/plugins/bootstrap/css/bootstrap.min.css.map HTTP/1.1" 404 1909
Not Found: /portal/dashboard/assets/images/flags/us.svg
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
Not Found: /portal/dashboard/assets/images/flags/jp.svg
Not Found: /portal/dashboard/assets/images/flags/es.svg
Not Found: /portal/dashboard/assets/images/flags/bl.svg
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/favicon.ico
"GET /portal/dashboard/favicon.ico HTTP/1.1" 404 2615
Not Found: /favicon.ico
Not Found: /
"GET / HTTP/1.1" 404 2443
Not Found: /
"GET / HTTP/1.1" 404 2443
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/themes/dark.png
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
Not Found: /portal/dashboard/assets/images/themes/rtl.png
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/us.svg
Not Found: /portal/dashboard/assets/images/flags/es.svg
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/bl.svg
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/themes/rtl.png
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/es.svg
Not Found: /portal/dashboard/assets/images/flags/us.svg
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/themes/dark.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
Not Found: /portal/dashboard/assets/images/flags/jp.svg
Not Found: /portal/dashboard/assets/images/flags/bl.svg
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/themes/rtl.png
Not Found: /portal/dashboard/assets/images/flags/es.svg
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
Not Found: /portal/dashboard/assets/images/flags/us.svg
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/themes/dark.png
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/bl.svg
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
Not Found: /
"GET / HTTP/1.1" 404 2443
Not Found: /portal/dashboard/project-ticket.html
"GET /portal/dashboard/project-ticket.html HTTP/1.1" 404 2639
Not Found: /portal/dashboard/project-ticket-details.html
"GET /portal/dashboard/project-ticket-details.html HTTP/1.1" 404 2663
Not Found: /portal/dashboard/page-profile.html
"GET /portal/dashboard/page-profile.html HTTP/1.1" 404 2633
Not Found: /portal/dashboard/project-ticket-details.html
"GET /portal/dashboard/project-ticket-details.html HTTP/1.1" 404 2663
Not Found: /portal/dashboard/login.html
"GET /portal/dashboard/login.html HTTP/1.1" 404 2612
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
Not Found: /portal/dashboard/assets/images/themes/dark.png
Not Found: /portal/dashboard/assets/images/themes/rtl.png
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
Not Found: /portal/dashboard/assets/images/flags/bl.svg
Not Found: /portal/dashboard/assets/images/flags/us.svg
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
Not Found: /portal/dashboard/assets/images/flags/es.svg
Not Found: /portal/dashboard/assets/images/flags/jp.svg
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/us.svg
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
Not Found: /portal/dashboard/assets/images/themes/dark.png
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
Not Found: /portal/dashboard/assets/images/themes/rtl.png
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/es.svg
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/flags/jp.svg
Not Found: /portal/dashboard/assets/images/flags/bl.svg
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
Not Found: /
"GET / HTTP/1.1" 404 2443
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2494
Not Found: /
"GET / HTTP/1.1" 404 2443
Not Found: /
"GET / HTTP/1.1" 404 2443
Not Found: /portal/dashboard/login.html
"GET /portal/dashboard/login.html HTTP/1.1" 404 2612
Not Found: /portal/dashboard/assets/images/flags/us.svg
"GET /portal/dashboard/assets/images/flags/us.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/flags/es.svg
"GET /portal/dashboard/assets/images/flags/es.svg HTTP/1.1" 404 2660
Not Found: /portal/dashboard/assets/images/themes/default.png
"GET /portal/dashboard/assets/images/themes/default.png HTTP/1.1" 404 2678
Not Found: /portal/dashboard/assets/images/flags/jp.svg
Not Found: /portal/dashboard/assets/images/xs/avatar3.jpg
Not Found: /portal/dashboard/assets/images/xs/avatar5.jpg
"GET /portal/dashboard/assets/images/xs/avatar5.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar2.jpg
"GET /portal/dashboard/assets/images/xs/avatar3.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/themes/rtl.png
"GET /portal/dashboard/assets/images/themes/rtl.png HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/xs/avatar4.jpg
"GET /portal/dashboard/assets/images/xs/avatar2.jpg HTTP/1.1" 404 2666
"GET /portal/dashboard/assets/images/flags/jp.svg HTTP/1.1" 404 2660
"GET /portal/dashboard/assets/images/xs/avatar4.jpg HTTP/1.1" 404 2666
Not Found: /portal/dashboard/assets/images/themes/dark.png
"GET /portal/dashboard/assets/images/themes/dark.png HTTP/1.1" 404 2669
Not Found: /portal/dashboard/assets/images/flags/bl.svg
"GET /portal/dashboard/assets/images/flags/bl.svg HTTP/1.1" 404 2660
"GET /static/assets/images/user.png HTTP/1.1" 404 1825
